cli_reference/openshift_developer_cli/creating-a-project-in-odo.adoc

```diff
@@ -0,0 +1,31 @@
+[id="creating-a-project"]
+= Creating a project
```

Comment:
This document feels out of place and I think it should instead be in each user case / document (deploying components, deploying multiple components).